simulation/traffic_simulator/test/src/traffic_lights/test_traffic_light_manager.cpp

```diff
@@ -63,20 +75,15 @@ TEST(TrafficLightManager, setColor)
   }
 }
 
-TEST(TrafficLightManager, setArrow)
+/**
+ * @note Test basic functionality. Test adding a bulb to a specific traffic light with all parameters specified.
+ */
+TEST_F(TrafficLightManagerTest, getTrafficLights_setArrow)
 {
-  const auto node = std::make_shared<rclcpp::Node>("setArrow");
-  std::string path =
-    ament_index_cpp::get_package_share_directory("traffic_simulator") + "/map/lanelet2_map.osm";
-  geographic_msgs::msg::GeoPoint origin;
-  origin.latitude = 35.61836750154;
-  origin.longitude = 139.78066608243;
-  const auto hdmap_utils_ptr = std::make_shared<hdmap_utils::HdMapUtils>(path, origin);
-  traffic_simulator::TrafficLightManager manager(hdmap_utils_ptr);
+  using Color = traffic_simulator::TrafficLight::Color;
+  using Status = traffic_simulator::TrafficLight::Status;
+  using Shape = traffic_simulator::TrafficLight::Shape;
```

Comment:
I think this should be moved to the fixture class

Reply:
I will just move usings to the top of the file